lib/loadable_config.rb

```diff
@@ -39,25 +39,42 @@ def self.attributes(*attrs, type: :string, optional: false)
   end
 
   def initialize
-    unless File.exist?(self.class._config_file)
-      raise RuntimeError.new("Cannot configure #{self.class.name}: configuration file '#{self.class._config_file}' missing")
+    prefix = LoadableConfig::Options.config_path_prefix || ''
```

Comment:
Would it make sense to build the defaults into the accessors, so that they could be used in multiple places?